components/progress-bar/script.js

```diff
@@ -10,6 +10,24 @@ import template from './template.html';
 
 const BaseComponent = components.BaseComponent;
 
+/**
+ * Check if a value is a number and if not - log an error
+ * @param {string} propName - the name of the property that needs to be validated
+ * @param {any} value
+ * @returns {boolean} - true if it is a number or a string that can be cast to number, false if not
+ */
+function isAPositiveNumberValidation(propName, value) {
```

Comment:
This is a nice candidate for util method so we can use it for the rest components :D (just saying :D)

Reply:
Added to the components library.